app/lib/theme/EmptyObjectFieldTemplate.tsx

```diff
@@ -0,0 +1,9 @@
+import { ObjectFieldTemplateProps } from "@rjsf/core";
+
+const EmptyObjectFieldTemplate: React.FC<ObjectFieldTemplateProps> = (
```

Comment:
Not quite sure I understand what this template does, it might need some comments